docs/proposals/27-project-group.md

```diff
@@ -0,0 +1,197 @@
+
+At the moment, [such a scenario is enabled](https://gardener.cloud/docs/guides/security-and-compliance/regional-restrictions/) by Gardener operators manually tainting seeds and allowing projects of said users to set tolerations for their shoots.
+
+A problem arises when such users want not only one, but multiple projects to be restricted to the same private seed(s). This would result in additional work for the Gardener operators to repeatedly set the same settings to the different projects of the user.
+
+The work of Gardener operators should be decreased by this enhancement. There is still some support required by them, because it should not be allowed for any arbitrary user to "reserve" a seed by themselves. It shall still be the decision of Gardener operators to decide if a user is allowed to have such a private seed.
+
+To summarize, there are two scenarios which need to be systematically enabled:
+
+1. Enable the assured scheduling of shoot control planes to specific geographical regions
+2. Let users have the opportunity to have a "private seed" (enabled by Gardener operators)
+
+Both scenarios are currently already technically possible, but require strong involvement of Gardener operators.
+
+Also users can easily ignore those restrictions, by simply setting the `.spec.seedSelector` of a shoot to match a seed without the taint / outside the required geographical location.
+
+In order to enforce something like this, a new resource called [`SeedBinding`s](#seedbinding) will be introduced.
+
+Furthermore, [`ProjectGroup`s](#projectgroup) should decrease the work of Gardener operators by letting users group different projects into a single project group, where seed bindings will automatically be shared among them.
+
+### Goals
+
+The overarching goal of this GEP is to lay the foundational work to enable homogeneous sovereign cloud scenarios.
+
+- Enable projects to use some seeds exclusively by using the Taints and Tolerations feature ([`SeedBinding`s](#seedbinding))
+- Enhance projects so that shoot control planes can be restricted to certain geographical regions ([`SeedBinding`s](#seedbinding))
+- Share those, so called, seed bindings to different projects that logically belong together in a concept called [`ProjectGroup`s](#projectgroup)
+- Introduce new role for project group administrators, that are allowed to add projects to a project group
+
+### Non-Goals
+
+- Let users exclusively bind seeds on their own
+
+## Proposal
+
+#### ProjectGroup
+
+A ProjectGroup API will be introduced that abstracts certain information over one or more projects. With this, it will become possible to homogeneously share resources between projects.
+
+Such a ProjectGroup manifest looks like this:
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: ProjectGroup
+metadata:
+  name: group1
+spec:
+  namespace: gardenprojectgroup-group1
+  owner:
+    apiGroup: rbac.authorization.k8s.io
+    kind: User
+    name: john.doe@example.com
```

Comment:
Is it really only one owner or could it be multiple?

Reply:
The `Project` resource has the following comment in the code:

```
// TODO: Remove this field in favor of the `owner` role in `v1`.
```

Do we really want to introduce this technical debt here or should we already start with the proper approach?